app/controllers/admin_posts_controller.rb

```diff
@@ -14,7 +14,7 @@ def index
     @admin_posts ||= AdminPost
     if params[:language_id].present? && (@language = Language.find_by(short: params[:language_id]))
       @admin_posts = @admin_posts.where(language_id: @language.id)
-      @tags = AdminPostTag.joins(:admin_posts).where(admin_posts: { language_id: @language.id })
+      @tags = AdminPostTag.distinct.joins(:admin_posts).where(admin_posts: { language_id: @language.id }).order(:name)
     else
       @admin_posts = @admin_posts.non_translated
       @tags = AdminPostTag.order(:name)
```

Comment:
I know the issue specifically mentions translated posts, but should the same `distinct` and `order` logic apply here as well?

Reply:
Admin post tag names have a uniqueness validation on them, so I'm not sure if `distinct` would add anything here. Adding a `joins(:admin_posts)` might not be a terrible idea to get rid of zero-use tags, but it is wandering a bit away from the original issue.